coding-intro/coding-intro-core/core-concepts/conditions-and-loops-two.md

```diff
@@ -0,0 +1,36 @@
+---
+author: lior-bd
+
+type: normal
+
+category: must-know
```

Comment:
Any insight ending in a question must be a discussion insight (this allows us to render special elements for discussion interactions)